e3nn/message_passing.py

```diff
@@ -0,0 +1,22 @@
+import torch
+from torch_geometric.nn import MessagePassing
```

Comment:
add `torch-geometric` as dependency? @bkmi

Reply:
I suppose we could have it as an extra?

It seems like a lot to add for this one single use.